src/shared/condition.c

```diff
@@ -556,18 +557,26 @@ static int condition_test_needs_update(Condition *c) {
                 return true;
```

Comment:
Shouldn't this also say something? It seems like an non-absolute param here is config error. Or actually, it seems better to reject such paths at parsing time.